### PR TITLE
feat: prepare template datamodel

### PR DESCRIPTION
**Pull request by tkuzynow**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/main/resources/db/changelog/changeset/0016_add_data_protection_attributes/addDataProtectionAttributes.sql
Comment (SheepFromHeaven) [inline src/main/resources/db/changelog/changeset/0016_add_data_protection_attributes/addDataProtectionAttributes.sql:1]: Should these be initially NULL as they are not nullable?

**Review by SheepFromHeaven — DISMISSED** [review]

File: src/main/resources/db/changelog/changeset/0016_add_data_protection_attributes/addDataProtectionAttributes.sql
Comment (tkuzynow) [reply src/main/resources/db/changelog/changeset/0016_add_data_protection_attributes/addDataProtectionAttributes.sql:1]: they may be null, as the feature is not available in all instances

File: services/consultingtypeservice.yaml [hunk 1]
```diff
@@ -198,9 +295,6 @@ components:
             slug:
               type: string
               example: part-of-url
-            languageFormal:
```
Comment (SheepFromHeaven) [inline services/consultingtypeservice.yaml:201]: Why was this removed O.o. Isn'T this still needed?

**Review by SheepFromHeaven — DISMISSED** [review]

File: services/consultingtypeservice.yaml (hunk 1 above)
Comment (tkuzynow) [reply services/consultingtypeservice.yaml:201]: it's consistent with current implementation of consulting type service, where this has been removed. It's used, but not required in ExtendedConsultingTypeResponseDTO

**Review by SheepFromHeaven — APPROVED** [review]
